### PR TITLE
add details about tools debugging

### PR DESCRIPTION
**Pull request by benlower**

### CODE REVIEW

File: docs/src/content/docs/sdk.mdx [hunk 1]
```diff
@@ -212,6 +212,25 @@ state.addEventListener('ultravoxExperimentalMessage', (msg) => {
 session.leaveCall();
 ```
 
+#### Debug Message:  Tool Call
```
Comment (mdepinet) [inline docs/src/content/docs/sdk.mdx:215]: Can we add a warning against relying on the structure or content of these messages please?

File: docs/src/content/docs/sdk.mdx [hunk 2]
```diff
@@ -212,6 +212,25 @@ state.addEventListener('ultravoxExperimentalMessage', (msg) => {
...
+When the agent invokes a tool, the message contains the function, all arguments, and an invocation ID:
+```bash
+LLM response: Tool calls: [FunctionCall(name='createProfile', args='{"firstName":"Ron","lastName":"Burgandy","organization":"Fixie.ai","useCase":"creating a talking AI news reporter"}', invocation_id='call_D2qQVS8OQc998aMEw5PRa9cF')]
+```
+
+#### Debug Message:  Tool Call Result
+When the tool call completes, the message contains an array of messages. The first is the tool call. The second is the tool result. These messages are also available via [List Call Messages](../api/calls/#list-call-messages)
```
Comment (mdepinet) [inline docs/src/content/docs/sdk.mdx:222]: This isn't actually quite right.  The model can invoke multiple tools at once.  This message will have all the calls followed by all the results.  (But I also think this is more information than necessary.)

**Review by mdepinet — APPROVED** [review]

File: docs/src/content/docs/sdk.mdx (hunk 1 above)
Comment (benlower) [reply docs/src/content/docs/sdk.mdx:215]: yessir

File: docs/src/content/docs/sdk.mdx (hunk 2 above)
Comment (benlower) [reply docs/src/content/docs/sdk.mdx:222]: ok i am tweaking a bit to be more correct

**Pull request merged**